app/Policies/LanguageLinePolicy.php

```diff
@@ -15,9 +15,9 @@ class LanguageLinePolicy
      *
      * @param \App\Models\User $user
      *
-     * @return bool|\Illuminate\Auth\Access\Response
+     * @return bool
      */
-    public function viewAny(User $user)
+    public function viewAny(User $user): bool
```

Comment:
Laravel `make:policy` doesn't bother on stating `|\Illuminate\Auth\Access\Response`. So why should we?!